packages/BackToTopButton/index.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <button
+    :class="[$style.float, { show: isShown }]"
+    @click="scrollToTop"
+    v-html="require('~/assets/ic-arror-top.svg')"
+  />
```

Comment:
Why not just wrap existed Button component for this? I think to add a shape attribute for the round button is work.

```diff
 <Button
   style-type="default"
+  shape="round"
 >
  <img slot="icon" src="~/assets/ic-arror-top.svg">
 </Button>
```

Reply:
The size of this float button is quite different from existed button. If we're going to use the existed `Button` component, we'll need to add 2 more options: `small`, `round`. So the usage would be like:
```vue
  <Button
    :class="[$style.float, { show: isShown }]"
    @click="scrollToTop"
    size="small"
    round
  >
    <span
      :class="$style.icon"
      slot="icon"
      v-html="require('~/assets/ic-arror-top.svg')"
    />
  </Button>
```
and overwrite the size of icon: 
```
.icon {
  width: $fs-16;
  height: $fs-16;
}
```

Reply:
You can put the `isShown` flag out of Button component, like:

```jsx
<div :class="[{show: isShown}]">
  <Button round />
</div>
```

Then only one option which needs to add to the Button.

Overwriting icon size is ok to me, but I prefer to specify the px directly to the width and height.